### PR TITLE
[TizenAppControl] Add SetAutoRestart method

### PR DESCRIPTION
**Pull request by hjhun**
The app_control_set_auto_restart() is supported since tizen 7.0.
The application can set the auto restart option at runtime using the function.
To use the function, the application has to be signed using the platform level certificate.

### CODE REVIEW

File: tools/generate_sysroot.py [hunk 1]
```diff
@@ -137,10 +137,10 @@ def generate_sysroot(sysroot: Path, api_version: float, arch: str, quiet=False):
   else:
     sys.exit('Unknown arch: ' + arch)
 
-  base_repo = 'http://download.tizen.org/snapshots/tizen/{}-base/latest/repos/standard/packages'.format(
-    api_version)
-  unified_repo = 'http://download.tizen.org/snapshots/tizen/{}-unified/latest/repos/standard/packages'.format(
-    api_version)
+  base_repo = 'http://download.tizen.org/snapshots/TIZEN/Tizen-{}/Tizen-{}-Base/latest/repos/standard/packages'.format(
```
Comment (swift-kim) [inline tools/generate_sysroot.py:140]: What's the difference between the directories `TIZEN` and `tizen`? Is `tizen` deprecated?

File: flutter/shell/platform/tizen/channels/app_control.cc [hunk 2]
```diff
@@ -369,4 +369,12 @@ AppControlResult AppControl::Reply(AppControl* reply,
                                              result_e);
 }
 
+AppControlResult AppControl::SetAutoRestart(bool enabled) {
+  if (enabled) {
+    return app_control_set_auto_restart(handle_);
```
Comment (swift-kim) [inline flutter/shell/platform/tizen/channels/app_control.cc:374]: Our target API level is Tizen 5.5. Tizen 7.0-specific API cannot be used in the embedder (especially if it's an internal API).

Can't you just call `app_control_set_auto_restart` using FFI in your app?

Comment (hjhun) [reply flutter/shell/platform/tizen/channels/app_control.cc:374]: The robot platform team wants to use this API to set/unset auto restart at runtime.
It's a requirement from Robot Platform team.
I'll discuss it with the developer again.
Thank you for reviewing this PR.

File: tools/generate_sysroot.py (hunk 1 above)
Comment (hjhun) [reply tools/generate_sysroot.py:140]: tizen directory was deprecated.

Currently, the build system(quickbuild) is uploading new binaries in 'Tizen' directory.

**Pull request closed without merging**